doc/source/user_guide/basics.rst

```diff
@@ -1812,8 +1812,8 @@ For information on key sorting by value, see :ref:`value sorting
 By values
 ~~~~~~~~~
 
-The :meth:`Series.sort_values` method is used to sort a `Series` by its values. The
-:meth:`DataFrame.sort_values` method is used to sort a `DataFrame` by its column or row values.
+The :meth:``Series.sort_values`` method is used to sort a ``Series`` by its values. The
```

Comment:
These types of references (e.g., :meth:\`Series.sort_values\`) actually require single rather than double backticks, this is probably why the doc build is failing

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#xref-syntax

Reply:
Hi @dsaxton , I've removed many unnecessary backticks 
now I cann't find problems that are failing the test. 
can you tell me what other problems I'd made in the commits